daemons/fenced/cts-fence-helper.c

```diff
@@ -125,10 +125,14 @@ st_callback(stonith_t * st, stonith_event_t * e)
         crm_exit(CRM_EX_DISCONNECT);
     }
 
-    crm_notice("Operation %s requested by %s %s for peer %s.  %s reported: %s (ref=%s)",
-               e->operation, e->origin, e->result == pcmk_ok ? "completed" : "failed",
-               e->target, e->executioner ? e->executioner : "<none>",
-               pcmk_strerror(e->result), e->id);
+    crm_notice("Operation '%s' targeting %s by %s for %s: %s (exit=%d, ref=%s)",
+               ((e->operation == NULL)? "unknown" : e->operation),
+               ((e->target == NULL)? "no node" : e->target),
+               ((e->executioner == NULL)? "any node" : e->executioner),
+               ((e->origin == NULL)? "unknown client" : e->origin),
```

Comment:
not giving an executioner is valid so "any node" as kind of natural language is fine.
but other things like not giving a target definitely denote something invalid.
so question is if things that are fishy shouldn't stick out a bit more like "<invalid node>".

Reply:
Actually some ops don't set a target (mainly meta-data iirc, but I think monitor sometimes doesn't).